recipes/sip/bld.bat

```diff
@@ -0,0 +1,8 @@
+%PYTHON% configure.py --sysroot=%PREFIX% --bindir=%LIBRARY_BIN%
```

Comment:
Huh, for some reason I thought they should live in `%LIBRARY_INC%`. Though you are definitely right that this is not the case with Python. Will trust your judgement here.